src/components/organisms/OakHeader/OakHeader.tsx

```diff
@@ -0,0 +1,191 @@
+                    src={authorImageSrc}
+                    $zIndex={"in-front"}
+                  />
+                  <OakBox>
+                    <OakHeading tag="h3" $font={"heading-7"}>
+                      {authorName}
+                    </OakHeading>
+                    <OakP>{authorTitle}</OakP>
+                  </OakBox>
+                </OakFlex>
+                <OakP $mb={["space-between-l", "space-between-none"]}>
+                  {subHeadingText}
+                </OakP>
+              </OakFlex>
+            </OakFlex>
+
+            <OakFlex
+              $position={"relative"}
+              $minHeight={[
+                "all-spacing-18",
+                "all-spacing-21",
+                "all-spacing-21",
+              ]}
+              $alignItems={"center"}
+              $justifyContent={"center"}
+              $mb={[
+                "space-between-none",
+                "space-between-m2",
+                "space-between-none",
+              ]}
+            >
+              <OakImage
+                $transform={"rotate(-2.025deg)"}
+                src={heroImageSrc}
+                width={100}
+                $minHeight={[
+                  "all-spacing-19",
+                  "all-spacing-20",
+                  "all-spacing-20",
+                ]}
+                $minWidth={[
+                  "all-spacing-19",
+                  "all-spacing-22",
+                  "all-spacing-21",
+                ]}
+                alt={`${headingTitle} hero image`}
+                $zIndex={"in-front"}
+              />
+              <OakFlex
+                $position={"absolute"}
+                $top={"all-spacing-0"}
+                $bottom={"all-spacing-0"}
+                $width={["all-spacing-20", "all-spacing-23", "all-spacing-23"]}
+              >
+                <OakImage
+                  src={
+                    "https://res.cloudinary.com/oak-web-application/image/upload/v1699887218/ui-graphics/loopdown.svg"
+                  }
+                  alt="loopdown ui-graphic"
+                  $position={"absolute"}
+                  $height={"100%"}
+                />
+              </OakFlex>
+            </OakFlex>
+            <OakFlex
+              $alignSelf={"flex-start"}
+              $ml={"space-between-m"}
+              $pt={"inner-padding-xl"}
+              $pb={[null, "inner-padding-xl"]}
+              $display={["block", "block", "none"]}
+            >
+              {breadcrumbs}
+            </OakFlex>
+          </OakFlex>
+        </OakMaxWidth>
+      </OakBox>
+    </OakThemeProvider>
+  );
+};
+
+/**
+ *
+ * Oak Heading component with hero image, can be used for blog posts, articles, and other content pages.
+ *
+ * This Oak component can be passed the OWA breadcrumbs component as a prop. Currently this component does not exist in the component library.
+ */
+export const OakHeader = styled(UnstyledComponent)`
```

Comment:
I worry this name is quite generic, in the designs it's `OakHeaderHero` which seems a bit better?

Reply:
✅